### PR TITLE
Clean the test csprojs to not set default properties.

### PR DESCRIPTION
**Pull request by Priya91**
cc @joperez

### CODE REVIEW

**Comment by davidsh** [comment]
Why are we doing this?  It would help others (me included) understand why we should do this.

**Comment by Priya91** [comment]
@davidsh These values are set in [dir.props](https://github.com/dotnet/corefx/blob/master/dir.props). Each individual test project need not configure them again.

File: src/System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj [hunk 1]
```diff
@@ -6,19 +6,7 @@
     <ProjectGuid>{0febe054-68ac-446f-b999-9068736d3cec}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AssemblyName>System.Private.Uri.Tests</AssemblyName>
-    <!--
-      Until we get first class support for NS1.7 and Netcoreapp1.1
-      we need to hard code $(TestTFM) and $(TestNugetTargetMoniker)
-      in the project file.
-    -->
-    <TestTFM>netcoreapp1.1</TestTFM>
-    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```
Comment (joperezr) [inline src/System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj]: ><NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker> [](start = 4, length = 66)

I think this one we might want to keep.

File: src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.csproj [hunk 2]
```diff
@@ -10,20 +10,8 @@
     <RootNamespace>System.Xml.XmlSerializer.Tests</RootNamespace>
     <AssemblyName>System.Xml.XmlSerializer.Tests</AssemblyName>
     <ProjectGuid>{4050F1D1-1DD2-4B48-A17B-E3F90DD18C4B}</ProjectGuid>
-    <!--
-        Until we get first class support for NS1.7 and Netcoreapp1.1
-        we need to hard code $(TestTFM) and $(TestNugetTargetMoniker)
-        in the project file.
-      -->
-      <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```
Comment (joperezr) [inline src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.csproj]: > <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker> [](start = 5, length = 67)

Same here

File: src/System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj (hunk 1 above)
Comment (Priya91) [inline src/System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj]: fixed.

File: src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.csproj (hunk 2 above)
Comment (Priya91) [inline src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.csproj]: fixed.

**Comment by Priya91** [comment]
@dotnet-bot test this please

**Comment by Priya91** [comment]
@dotnet-bot test  Innerloop CentOS7.1 Debug Build and Test

**Comment by Priya91** [comment]
@dotnet-bot test  Innerloop CentOS7.1 Release Build and Test

**Pull request merged**